### ISSUE TITLE
Using this action in a workflow_run.completed handler

### ISSUE THREAD

**Issue by chaimmintz** [Kesin11/actions-timeline#157]
Is it possible to have this run in a workflow that gets launched as a `workflow_run.completed`, action like:

```
on:
  workflow_run:
    workflows:
      - 'myworkflow'
    types:
      - completed
```

In this case, it should not use *itself* as the workflow run, but instead use the workflow run that triggered it (i.e. fetch that run and post the results to that workflow run summary). This would allow us to have it done in a completely separate workflow that works for many other workflows and also doesn't look like it's part of the other workflows.

That is how [int128/datadog-actions-metrics@v1](https://github.com/int128/datadog-actions-metrics/blob/38af2cbdb4d12908921b26d036ecd59d46d58c8b/src/workflowRun/handler.ts#L18) is implemented to work and it's super useful.

```
  core.info(`Got workflow run ${e.action} event: ${e.workflow_run.html_url}`)
  if (e.action === 'completed') {
    <do something useful with e.workflow_run.id>
  }
```

**Comment by Kesin11** [comment]
@chaimmintz 
It is an interesting idea, and I agree that it is convenient to separate the actions-timeline workflow from the actual workflow. Unfortunately, I don't know how to update the job summary from another workflow.

The int128/datadog-actions-metrics action seems to only send metrics to Datadog. So to implement that idea we need to find a way to update the job summary.

**Comment by Kesin11** [comment]
I have an idea to use check runs instead of job summaries. Check runs also accept markdown and can be created via the [API](https://docs.github.com/en/rest/checks/runs?apiVersion=2022-11-28#create-a-check-run). This approach might allow us to create new check runs that display actions-timeline Gantt diagrams from another workflow.
What do you think?

**Comment by chaimmintz** [comment]
Yeah, check runs would be fine. (For me even if it is just on the summary of this run and not the original run it would still be ok since I can figure out how to find it when needed.)

**Comment by Kesin11** [comment]
@chaimmintz 
I'll try to using checks runs when I have free time.
If you create a pull-request, I will be happy to review it.
